src/Shared/Recruit.Vacancies.Client/Infrastructure/EventHandlers/VacancyClosedEventHandler.cs

```diff
@@ -46,10 +46,22 @@ public class VacancyClosedEventHandler : INotificationHandler<VacancyClosedEvent
 
         public async Task Handle(VacancyClosedEvent notification, CancellationToken cancellationToken)
         {
-            _logger.LogInformation("Deleting LiveVacancy {vacancyReference} from query store.", notification.VacancyReference);
-            await NotifyFaaVacancyHasClosed(notification);
-            await _queryStore.DeleteLiveVacancyAsync(notification.VacancyReference);
-            await CreateClosedVacancyProjection(notification.VacancyId);
+            var vacancy = await _repository.GetVacancyAsync(notification.VacancyId);
+            var naturallyClosedVacancy = vacancy.ClosureReason == ClosureReason.Auto ||
+                                                  vacancy.ClosureReason == ClosureReason.Manual;
+            if (naturallyClosedVacancy)
```

Comment:
Do we need this condition? Shouldn't all closed vacancies be removed from FAA?

Reply:
@dev4valtech suggested that the normal logic will not be required.

Reply:
@shomavg, we cannot assume from ClosureReason that the vacancy was never live. So @chrisjensenuk is right. Apologies but you will have to revert the changes in the handler.

Reply:
@shomavg publishing VacancyClosedEvent is not a good idea. There are two options, you can update the dashboard directly by `await _dashboardService.ReBuildDashboardAsync(vacancy.EmployerAccountId);` or raise a completely new event that will by handled by `UpdateEmployerDashboardOnChange`